integration-tests/bats/helper/gen_keys.bash

```diff
@@ -0,0 +1,38 @@
+python3 -c '
```

Comment:
Does this need new python dependencies installed when `bats` runs? Is it better/possible to use `go/libraries/jwtauth/gen_key/main.go` instead?